flink-runtime/src/main/java/org/apache/flink/runtime/io/network/bufferprovider/GlobalBufferPool.java

```diff
@@ -63,8 +64,13 @@ public GlobalBufferPool(int numBuffers, int bufferSize) {
 
 		for (int i = 0; i < numBuffers; i++) {
 			try {
-				byte[] buf = new byte[bufferSize];
-				buffers.add(new MemorySegment(buf));
+				// TODO if (config) {
```

Comment:
Thanks for pointing that out. They are currently used in the memory manager of the task manager but not for the GlobalBufferPool. I probably want to fix that.